docs/builders/ephemeris/horizons-gui.md

```diff
@@ -0,0 +1,83 @@
+
+## Horizons log messages
+This is a log that only displays messages related to your input in this tool and the download of the data. When using this tool there are a few different issues that you can encounter and they will be explained in more detail below. All of the issues you will encounter will be detailed in this log and the messages will guide you towards a fix. If OpenSpace fails with getting this helpful information there will be an error file stored at the specified location in the **File Path** field. This file might contain some useful information about what went wrong and how it can be fixed.
+
+
+# Issues and Solutions
+There are a few different things that can go wrong while using this tool. Most of the issues are related to the input and can be fixed with some adjustments. The particular parameters that need to be adjusted will be highlighted in red and there will be some messages in the Horizons log that can give you tips on how to fix it.
+
+## No matching target
+The target you searched for could not be found. Check that the name has been spelt correctly and you can also add a <code>*</code> at the end of the name to widen your search. If you still cannot find what you are looking for then it is possible that there might be an alternative name for it in the Horizons database or that it does not exist.
+
+## Multiple matching targets
+The target you searched for found multiple matches in the Horizons database. There should be a added drop down menu close to the **Target Body** field where you can select one of the matches. If none of them is what you were looking for, you can keep the first item in the list selected and write a new name to search for.
+
+## Target and Observer are the same
+This issue only happens when the **Observer Table** format is selected and happens when the target and the observer are the same object. This can be fixed by either changing the observer or the target to something else. If the **Vector Table** format is selected, then this is allowed even if it does not give any useful results (all the data will be zero).
+
+## No matching observer
+The observer location you searched for could not be found. Check that the name has been spelt correctly and you can also add a <code>@</code> to the beginning of the name to widen your search. If you still cannot find what you are looking for then it is possible that there might be an alternative name for it in the Horizons database or that it does not exist.
+
+## Multiple matching observers
+The observer you searched for found multiple matches in the Horizons database. There should be a added drop down menu close to the **Observer Location** field where you can select one of the matches. If none of them is what you were looking for, you can keep the first item in the list selected and write a new name to search for.
```

Comment:
"The observer you searched for found multiple matches in the Horizons database."
->
"Multiple matches were found in the Horizons database for the observer you searched for."  ?